src/main/webapp/resources/js/components/ant.design/TextWithHelpPopover.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { Popover, Space, Typography } from "antd";
+import { QuestionCircleOutlined } from "@ant-design/icons";
+
+type Props = {
+  text: JSX.Element | string;
+  help: string;
+};
+
```

Comment:
Function comment

Reply:
This file was created then not used, removed in 44aa205fab05bfca261f80506b219e689745d0f7